docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+Getting started from Scratch
+----------------------------
+
+**Table of Contents**
+
+@@TOC@@
+
+### Intended Audience
+
+This guide is for people who want to craft a custom Kubernetes cluster.  If you
+can find an existing Getting Started Guide that meets your needs on [this
+list](README.md), then we recommend using it, as you will be able to benefit
+from the experience of others.  However, if you have specific IaaS, networking,
+configuration management, or operating system requirements not met by any of
+those guides, then this guide will provide an outline of the steps you need to
+take.  Note that it requires considerably more effort than using one of the
+pre-defined guides.
+
+This guide is also useful for those wanting to understand at a high level some of the
+steps that existing cluster setup scripts are making.
+
+### Prerequisites
```

Comment:
Gathering from the sections below, we should also clarify and list the other items needed. For example, something like the following (please revise for clarity/accuracy):

3. Depending on your environment and requirements, you need to design and implement:
     - Networking model
     - Security model
4. For this getting started guide, the following hardware and software are required (these requirements are later discussed in detail):
     - A machine running x86_64 Linux for each node in the cluster.
     - *Mention whats needed or the assumptions made WRT "a working physical network"*
     - Download, install, and configure the following binaries:
          - Kubernetes (includes etcd, Docker, and rkt)